proof/refine/AARCH64/Schedule_R.thy

```diff
@@ -249,9 +249,23 @@ lemma switchLocalFpuOwner_corres[corres]:
    apply (auto simp: current_fpu_owner_Some_tcb_at fpuOwner_asrt_cross)
   done
 
+(*FIXME FPU: move tcbFlag lemmas*)
+lemma tcbFlagToWord_bit:
+  "\<exists>n<64. tcbFlagToWord flag = bit n"
+  by (auto simp: tcbFlagToWord_def ex_nat_less_eq split: tcb_flag.splits simp del: bit_def)
+
+lemma FpuDisabled_in_tcbFlagMask[simp]:
```

Comment:
not sure what to do in such naming cases (capital starting letter): @lsf37 ?

Reply:
I was about to write that my gut feeling tends towards lower case, but looking at it for longer, if it was `None` we would not turn it into `none` if we wanted to refer to the constructor. So I think we should leave it.